articles/article-122-ascii.md

```diff
@@ -0,0 +1,36 @@
+<!-- BEGIN TITLE -->
+# Every letter is a number
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+If letters are represented in computers they are stored as numbers. To encode them every letter got its own decimal number between 0 and 127. The "H", for example, is encoded as the 72.
```

Comment:
Language: To encode them, every letter is assigned its own decimal number between 0 and 127.

Reply:
changed